.github/workflows/db.yml

```diff
@@ -34,20 +31,49 @@ jobs:
           DAG_CARGO_USER: ${{secrets.DAG_CARGO_USER}}
           DAG_CARGO_PASSWORD: ${{secrets.DAG_CARGO_PASSWORD}}
 
-  deploy-staging:
-    name: Deploy Staging DB
-    if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+  # this doesn't *do* anything, so commenting out until we can actually deploy to staging db from CI.
+  # deploy-staging:
+  #   name: Deploy Staging DB
+  #   if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+  #   runs-on: ubuntu-latest
+  #   steps:
+  #     - uses: actions/checkout@v2
+  #     - uses: actions/setup-node@v2
+  #       with:
+  #         node-version: 16
+  #     - uses: bahmutov/npm-install@v1 
+  #     # TODO: fork prod db
+  #     # TODO: Update staging schema!!!
+  #     - name: Test upload to staging
+  #       run: |
+  #         npm run build -w packages/client
+  #         echo "$(date --utc --iso-8601=seconds) web3.storage upload test" > ./upload-test-small
+  #         ./packages/w3/bin.js put ./upload-test-small --api https://api-staging.web3.storage --token ${{ secrets.STAGING_WEB3_TOKEN }}
+
+  release:
+    name: Release
```

Comment:
Shouldn't we check for `if: github.event_name == 'push' && github.ref == 'refs/heads/main'` like commented above?